solana-common/proto/instructions.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package sol.instructions.v1;
+
+message Instructions {
+  repeated Instruction instructions = 1;
+}
+
+message Instruction {
+  string program_id = 1;
+  repeated string accounts = 2;
+  string data = 3;
```

Comment:
If we keep instruction I think this one should be `bytes` directly, data is byte centric and it will make consumption for the IDL much easier after since it will read the byte directly and avoid the extra string to bytes decoding.